airq/app.py

```diff
@@ -81,6 +82,15 @@ def update_status(self, data):
             temp=self.strip_sensor_name(formatted_values[consts.KEY_TEMP]),
             co2=self.strip_sensor_name(formatted_values[consts.KEY_CO2])
         )
+
+    @rumps.clicked("Menu bar text...")
```

Comment:
Might be the perfect time to make this "Preferences..." and move that into there.  IMHO